tools/README.md

```diff
@@ -18,12 +18,17 @@ found in hardware_provisioning.sh
 ```
 DEPLOY_NAME=${DEPLOY_NAME:-cnftestbed}
 USE_RESERVED=${USE_RESERVED:-false}
-PACKET_FACILITY=${PACKET_FACILITY:-sjc1}
-PACKET_OS=${PACKET_OS:-ubuntu_16_04}
-MASTER_COUNT=${MASTER_COUNT:-1}
-MASTER_PLAN=${MASTER_PLAN:-m2.xlarge.x86}
-WORKER_COUNT=${WORKER_COUNT:-1}
-WORKER_PLAN=${WORKER_PLAN:-n2.xlarge.x86}
+STATE_FILE=${STATE_FILE:-$(pwd)/data/$DEPLOY_NAME/terraform.tfstate}
+NODE_FILE=${NODE_FILE:-$(pwd)/data/$DEPLOY_NAME/kubernetes.env}
+
+NODE_GROUP_ONE_NAME=${NODE_GROUP_ONE_NAME:-$DEPLOY_NAME-master}
+NODE_GROUP_TWO_NAME=${NODE_GROUP_TWO_NAME:-$DEPLOY_NAME-worker}
+NODE_GROUP_ONE_COUNT=${NODE_GROUP_ONE_COUNT:-1}
+NODE_GROUP_TWO_COUNT=${NODE_GROUP_TWO_COUNT:-1}
+FACILITY=${PACKET_FACILITY:-sjc1}
+NODE_GROUP_ONE_DEVICE_PLAN=${NODE_GROUP_ONE_DEVICE_PLAN:-m2.xlarge.x86}
```

Comment:
Is there any way to get away from hard coding the machine types?

Reply:
These are default variables, so can be overridden.